docs-ref-conceptual/python-sdk-azure-install.md

```diff
@@ -69,4 +69,27 @@ If you want to install `azure` from source:
 
     git clone git://github.com/Azure/azure-sdk-for-python.git
     cd azure-sdk-for-python
-    python setup.py install
\ No newline at end of file
+    python setup.py install
+
+## Install an older version with pip
+You can install an older version of `azure` by specifying 'azure==3.0.0' version details.
+```bash
+pip install azure==3.0.0 
+```
+## Check SDK installation details with pip
+You can check `azure` SDK installation location, version details etc.
+```bash
+pip show azure # Show installed version, location details etc.
+pip freeze     # Output installed packages in requirements format.
+pip list  # List installed packages, including editables.
+```
+## To uninstall with pip
+You can uninstall all Azure libraries in a single line using the `azure` meta-package.
+```bash
+pip uninstall azure 
+```
+> [!NOTE]
+> `pip uninstall azure`removes the `azure` meta-package but leaves the individual `azure-*` packages behind (and others, like `adal` and `msrest` ). It's a Python and pip limitation, for all packages that has dependencies, uninstalling the initial package does not uninstall the dependencies. To remove `azure-` and it's supporting packages, run the command `pip freeze | grep 'azure-' | xargs pip uninstall -y` (and then perform individual uninstalls for adal, msrest, and msrestazure).
```

Comment:
1. Instead of "limitation," recommend you call it an "aspect," and also a grammar fix (_have_ instead of _has_):

    An aspect of Python and pip is that for all packages that have dependencies, uninstalling the initial package does not uninstall the dependencies.

1. Another grammar issue - no apostrophe in "its" in this:

    To remove `azure-` and **its** supporting packages

Reply:
Done! Thanks.